doc/user/content/installation/install-on-gcp/_index.md

```diff
@@ -26,6 +26,11 @@ modules](https://github.com/MaterializeInc/terraform-google-materialize) to:
    module to deploy Materialize Operator and Materialize instances to the GKE
    cluster.
 
+Starting in v0.2.1 of Materialize on GCP Terraform, storage bucket versioning is
+disabled to facilitate cleanup of resources during testing. When running in
+production, versioning should be turned on with a sufficient TTL to meet any
+data-recovery requirements.
+
```

Comment:
In the referenced pr I did decide to enable versioning with a TTL of 7 days for non active versions. 

This is a middle ground that more easily allows one to test without versioning enabled while maintaining the preferred production setting.

Reply:
👍 will do a patch tomorrow.

Reply:
So ... apparently, "tomorrow" is "next week" 😄 .  
In any case, it's still disabled, yes? (I see the default is `false`) it's just that the ttl default value is set to 7.

Reply:
oh yeah, sorry I think I waffled on this a few times thanks for reading closely!!!